assignment5.txt

```diff
@@ -0,0 +1,13 @@
+1. What do each of the following mean? Include the description of the syntax, or an example. 
+	Passing by value: This means you pass the actual value of the variable into the function. An example is int i = 5;
```

Comment:
well, it means you pass a copy of the actual value so the original isn't modified. passing by reference means changes occur to the original.